docs/source/tutorials/hospital_admissions_model.qmd

```diff
@@ -179,11 +180,34 @@ I0 = InfectionInitializationProcess(
 
 # Generation interval and Rt
 gen_int = deterministic.DeterministicPMF(gen_int, name="gen_int")
-rtproc = process.RtRandomWalkProcess(
-    Rt0_dist=dist.TruncatedNormal(loc=1.2, scale=0.2, low=0),
-    Rt_transform=transformation.ExpTransform().inv,
-    Rt_rw_dist=dist.Normal(0, 0.025),
-)
+
+
+class MyRt(metaclass.RandomVariable):
```

Comment:
Model description above should be updated to reflect this.

Reply:
Addressed in 5869033